assets/prometheus-k8s/prometheus-k8s-service-monitor-web-console.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: web-console
+  labels:
+    k8s-app: web-console
+spec:
+  selector:
+    matchLabels:
+      k8s-app: web-console
+  namespaceSelector:
+    matchNames:
+    - this-namespace-is-automatically-replaced-at-runtime
```

Comment:
I'm pretty sure this should match the actual namespace in which the web console is installed